stdlib/public/SwiftShims/RefCount.h

```diff
@@ -988,6 +988,7 @@ class RefCounts {
   // First slow path of doDecrement, where the object may need to be deinited.
   // Side table is handled in the second slow path, doDecrementSideTable().
   template <PerformDeinit performDeinit>
+  SWIFT_ALWAYS_INLINE
```

Comment:
This is interesting. I feel like if it's always-inline it should probably be renamed, if we can. "always inline slow path" is an odd idea.